### PR TITLE
Code and javadoc cleanup, published stable 0.3.0

### PR DESCRIPTION
**Pull request by marcosalis**
```
com.teamwork:android-multiautocomplete:0.3.0
```

- **Code style:** removed a few Lint and compilation warnings in production code
- **Code style:** some project settings updated (line length, annotations wrapping)
- Updated **Javadocs** to latest class changes
- Using `com.github.panpf.bintray-publish:bintray-publish` fork of Novoda Bintray **publishing library** (the latter is not compatible with Gradle 6+)

### CODE REVIEW

File: multiautocomplete/src/test/java/com/teamwork/autocomplete/AutoCompleteAdapterTest.java [hunk 1]
```diff
@@ -24,6 +24,7 @@
 import static org.hamcrest.CoreMatchers.is;
 import static org.hamcrest.CoreMatchers.notNullValue;
 import static org.hamcrest.MatcherAssert.assertThat;
+import static org.mockito.ArgumentMatchers.any;
 import static org.mockito.Matchers.anyObject;
```
Comment (aidangrabe) [inline multiautocomplete/src/test/java/com/teamwork/autocomplete/AutoCompleteAdapterTest.java]: Unused import?

File: multiautocomplete/src/main/java/com/teamwork/autocomplete/adapter/BaseTypeAdapterDelegate.java [hunk 2]
```diff
@@ -51,13 +51,15 @@
...
+ * A typed adapter for the auto complete view, used to provide a layout, data binding and filter options for a single data type.
+ * It holds an {@link AutoCompleteViewBinder} and a {@link TokenFilter}.
  *
  * @author Marco Salis
  */
 // @ThreadSafe
-class BaseTypeAdapterDelegate<M> extends BaseAdapter implements TypeAdapterDelegate<M>, AutoCompleteTypeAdapter<M> {
+class BaseTypeAdapterDelegate<M>
+        extends BaseAdapter
+        implements TypeAdapterDelegate<M>, AutoCompleteTypeAdapter<M> {
```
Comment (aidangrabe) [inline multiautocomplete/src/main/java/com/teamwork/autocomplete/adapter/BaseTypeAdapterDelegate.java:62]: I get a nullability lint warning here that we can suppress I'd say

**Review by aidangrabe — APPROVED** [review]
Looks good 😄

File: multiautocomplete/src/main/java/com/teamwork/autocomplete/adapter/BaseTypeAdapterDelegate.java (hunk 2 above)
Comment (marcosalis) [reply multiautocomplete/src/main/java/com/teamwork/autocomplete/adapter/BaseTypeAdapterDelegate.java:62]: The only way to suppress it is to suppress any nullable problems in the whole class. A bit risky. We'll sort it out when we convert to Kotlin 👍

**Pull request merged**